ui/ui.go

```diff
@@ -0,0 +1 @@
+package ui
```

Comment:
Please add comments about why do we need this

Reply:
sure